### ISSUE TITLE
Index of Difficulty: Calculate using target height for Goal and Tunnel modes

### ISSUE THREAD

**Issue by bastianilso** [med-material/TunnelGoalFittsTests#31]
In the application we can calculate Index of Difficulty using this formula:
![image](https://user-images.githubusercontent.com/3967945/72238534-4ed7d180-35de-11ea-8198-667d65d37d60.png)

However, this works only for the Fitts Law test. For the Goal test, we should replace _W_ (target width) with _H_ (target height). This is column does not exist yet for each test result, so will need to be logged into the DB.

**Comment by bastianilso** [comment]
@hendrikknoche what was the formula for calculating the ID for the tunnel test?

**Comment by Adel-Akbi** [comment]
It's that one, when I changed the formula on the code, I've got negative values, is that possible? @hendrikknoche 

![formula](https://user-images.githubusercontent.com/45661981/72244217-14763080-35ee-11ea-83be-f64c130170fb.PNG)

**Comment by hendrikknoche** [comment]
yes, negative values are possible for 'incorrect' input. It's not possible to have distances that are equal to or smaller than half the width (in those case you would not have to move anywhere since you're already in the target). when D<=1/2W the logarithm returns 0 or smaller.
